src/components/button/button.scss

```diff
@@ -629,41 +630,49 @@
 :host([scale="s"]) a.content--slotted {
   @apply text-n2h;
 }
+
 // accommodate for transparent buttons not having borders
 :host([scale="s"][appearance="transparent"]) button.content--slotted,
 :host([scale="s"][appearance="transparent"]) a.content--slotted {
   --calcite-button-padding-x: theme("padding.2");
-  --calcite-button-padding-y: theme("padding.1");
 }
+
+:host([scale="s"]) button,
```

Comment:
nitpick: Ideally, both `button` and `a` have a class so we can just use that instead of the tag.

Reply:
Would this require quite a bit of refactoring, since button and a have been used throughout the file?

Reply:
yeah, feel free to ignore it.